Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_EditorEnhancements.cpp

```diff
@@ -1,7 +1,9 @@
 // Copyright Offworld Industries Ltd. All Rights Reserved.
 
 #include "OWI_EditorEnhancements.h"
+

```

Comment:
What code standard of ours requires an extra line break here?

Reply:
Order of includes is declaration header, engine/thridparty code and then our headers i think. A line break between the categories is expected too.